.gitignore

```diff
@@ -0,0 +1,15 @@
+*.pyc
+test.py
+*.gh
+.pytest_cache
+*__pycache__
+.coverage
+*.ipynb
+.ipynb_checkpoints
+.tox
+*.egg-info
+tox.ini
+/.cache
+/.vscode
+.eggs
+*.code-workspace
```

Comment:
Wow, this is quite a testament to all of the things we do with the code. If we exclude .gh files, should we also exclude .dyn files?

It also might just be worth clarifying in the README that this is a template just for new Python libraries it isn't intended for plugins or interface-specific libraries.

Reply:
I removed .gh to keep it generic.